std/range/package.d

```diff
@@ -8583,6 +8618,7 @@ public:
     }
     else static if (isForwardRange!R)
     {
+        import std.traits : isSafe;
```

Comment:
Does it matter that this import is in a `public:` scope of the struct `RefRange`?  I see the same being done previously in this module with the `Result` struct and its import of `anySatisfy`.